app/templates/lot.hbs

```diff
@@ -26,7 +26,7 @@
     {{#if model.value.zonedist1}}
       <ul class="lot-zoning-list">
         <li class="menu-text">Zoning District{{if (or model.value.zonedist2 model.value.spdist1) 's'}}:</li>
-        {{#if model.value.zonedist1}}<li><a target="_blank" href="https://www1.nyc.gov/site/planning/zoning/districts-tools/{{primaryzone1}}.page" class="button">{{fa-icon 'external-link-alt' transform='shrink-3 up-1'}} {{model.value.zonedist1}}</a></li>{{/if}}
+        {{#if (eq (eq model.value.zonedist1 "BPC") false)}}<li><a target="_blank" href="https://www1.nyc.gov/site/planning/zoning/districts-tools/{{primaryzone1}}.page" class="button">{{fa-icon 'external-link-alt' transform='shrink-3 up-1'}} {{model.value.zonedist1}}</a></li>{{/if}}
```

Comment:
Seems like there is an `unless` operator available in ember handlebars! So this would look like:
`{{#unless (eq model.value.zonedist1 "BPD")}}...{{/unless}}`